### ISSUE TITLE
Installing: purs executable not found

### ISSUE THREAD

**Issue by ChrisPenner** [purescript/purescript#2819]
Going through the install process: 

```bash
$ npm install -g purescript
$ npm install -g pulp bower
$ psc --version
0.10.7
$ pulp -v
Pulp version 11.0.0
$ pulp init
* ERROR: `purs` executable not found.
```

Not sure what's up, I had a really old version of purescript before, so I npm uninstalled both purescript and pulp before starting over and ended up with this error.

I ended up just `brew install purescript` and everything is working; but this is a potential painpoint
for new users so I figured I'd mention it.

Cheers! Thanks for all the work you do!

**Comment by garyb** [comment]
At the moment you'll need to install `purescript@next`. That'll be updated very soon, once we have 0.11.3 on npm! (`@next` is 0.11.1)

**Comment by ChrisPenner** [comment]
Just wanted to let you know that new users following the "getting started" guide won't properly "get started"; what you do now is up to you 😁 

Cheers!

**Comment by gfixler** [comment]
I followed [this setup tutorial](https://github.com/jdegoes/intro-to-ps) today at PureScriptConf, hitting errors at every step (lots of severely out-of-date things (haven't played in PS in a long time), and conflicting install stuff), but was able to solve them all, except the final step; when builing, I get that purs is missing. I tried installing `purescript@next`, but it didn't solve things for me.

**Comment by garyb** [comment]
`@next` is no longer necessary, as the published "normal" version is 0.11.3 now - what OS are you using? Maybe we don't provide a binary for whatever it is you're using. Were there no errors in the `npm install -g purescript` stage?

**Comment by MatrixMike** [comment]
Yes I like the 'expected' look and feel of PS based on my experience with Haskell etc but I have a really broken system as the installation instructions are baffling, ambiguous, incomplete and do not result in a working system. I can do tiny compiles 'by hand' but paths are 'up the creek' . I am gifted with plenty of patience but this is quite a test...    As a beginner I feel that other beginners would have given up by now and commented as indicated by the comments above.

**Comment by hdgarrood** [comment]
@matrixmike The issue here has been fixed. I just recently went through the guide with a beginner with no problems so I am not sure what you mean; I'm happy to help but I'll need more detail. Can you open a new issue on https://github.com/purescript/documentation please?

**Comment by rafiu007** [comment]
DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.

getting this error after running this 'npm install -g purescript '
